tests/ft_sh.vader

```diff
@@ -12,3 +12,6 @@ Execute (shellcheck):
   set ft=sh
   norm i#!/usr/bin/env bash
   AssertEqual neomake#makers#ft#sh#shellcheck()['args'], ['-fgcc']
+
+  norm ggcG# shellcheck shell=bash
```

Comment:
Hmm, yea. Either change all of them in an own commit or leave that out for now and we clean it up later.